src/mame/drivers/polepos.cpp

```diff
@@ -893,24 +887,27 @@ void polepos_state::polepos(machine_config &config)
 
 	namco_06xx_device &n06xx(NAMCO_06XX(config, "06xx", MASTER_CLOCK/8/64));
 	n06xx.set_maincpu(m_maincpu);
+	n06xx.chip_select_callback<0>().set("51xx", FUNC(namco_51xx_device::chip_select));
+	n06xx.rw_callback<0>().set("51xx", FUNC(namco_51xx_device::rw));
 	n06xx.read_callback<0>().set("51xx", FUNC(namco_51xx_device::read));
 	n06xx.write_callback<0>().set("51xx", FUNC(namco_51xx_device::write));
 	n06xx.read_callback<1>().set("53xx", FUNC(namco_53xx_device::read));
-	n06xx.read_request_callback<1>().set("53xx", FUNC(namco_53xx_device::read_request));
+	n06xx.chip_select_callback<1>().set("53xx", FUNC(namco_53xx_device::chip_select));
 	n06xx.write_callback<2>().set("52xx", FUNC(namco_52xx_device::write));
 	n06xx.write_callback<3>().set("54xx", FUNC(namco_54xx_device::write));
 
 	WATCHDOG_TIMER(config, "watchdog").set_vblank_count(m_screen, 16);   // 128V clocks the same as VBLANK
 
-	config.set_maximum_quantum(attotime::from_hz(6000));  /* some interleaving */
-
 	NVRAM(config, "nvram", nvram_device::DEFAULT_ALL_1);
 
 	TIMER(config, "scantimer").configure_scanline(FUNC(polepos_state::scanline), "screen", 0, 1);
 
 	LS259(config, m_latch); // at 8E on polepos
 	m_latch->q_out_cb<0>().set_inputline(m_maincpu, 0, CLEAR_LINE).invert();
-	m_latch->q_out_cb<1>().set(FUNC(polepos_state::iosel_w));
+	m_latch->q_out_cb<1>().set("51xx", FUNC(namco_51xx_device::reset));
+	m_latch->q_out_cb<1>().set("52xx", FUNC(namco_52xx_device::reset));
+	m_latch->q_out_cb<1>().set("53xx", FUNC(namco_53xx_device::reset));
+	m_latch->q_out_cb<1>().set("54xx", FUNC(namco_54xx_device::reset));
```

Comment:
The first three of these aren’t going to do anything useful – you need to use `.append` if you want to chain additional handlers.

Reply:
Ah good catch. Fixed in https://github.com/mamedev/mame/pull/6595.